setup.py

```diff
@@ -97,12 +97,6 @@ def load_deps(section):
     ],
     setup_requires=['pytest-runner'],
     tests_require=tests_require,
-    extras_require={
-        # jupyter-server is the new "infrastructure" of jupyter, in the Iguazio Jupyter we're still using an old version
-        # which uses the notebook-server. installing jupyter-server there is causing troubles (unwanted upgrade of
-        # tornado package, so we're installing jupyter-server only if explicitly requested by adding an extra
-        "jupyter-server": ["jupyter-server~=1.0"],
```

Comment:
worth keeping the extras for backwards compatibility. you can leave brackets empty